autoload/lsp_settings.vim

```diff
@@ -91,11 +91,42 @@ function! s:vim_lsp_installer(ft, ...) abort
     let l:servers += s:settings['_']
   endif
 
+  let l:default = get(g:, 'lsp_settings_filetype_' . a:ft, '')
+
   let l:name = get(a:000, 0, '')
   for l:conf in l:servers
     if !empty(l:name) && l:conf.command != l:name
       continue
     endif
+
+    if lsp_settings#get(l:conf.command, 'disabled', get(l:conf, 'disabled', 0))
```

Comment:
I don't like global default value `get(l:conf, 'disabled', 0)` since it should always be command name.

Reply:
I am not sure about this line, but it mirrors [autoload/lsp_settings.vim#L463](https://github.com/mattn/vim-lsp-settings/blob/52ae22b7800b132bb841fbfbed525297617df162/autoload/lsp_settings.vim#L463) right now.  Perhaps both need to be fixed?

Reply:
Ah, sorry. This is my fault.